dist/tools/travis-scripts/build_and_test.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e
+if [[ $BUILDTEST_MCU_GROUP ]]
+then
+	if [ "$BUILDTEST_MCU_GROUP" == "x86" ]
+	then
+		make -s -C ./examples/default info-concurrency
+		git rebase riot/master || git rebase --abort
+
+		./dist/tools/licenses/check.sh master --diff-filter=MR --error-exitcode=0 || exit
+		./dist/tools/licenses/check.sh master --diff-filter=AC || exit
+
+		./dist/tools/doccheck/check.sh master || exit
+
+		./dist/tools/externc/check.sh master || exit
+
+		# TODO:
+		#   Remove all but `master` parameters to cppcheck (and remove second
+		#   invocation) once all warnings of cppcheck have been taken care of
+		#   in master.
+		./dist/tools/cppcheck/check.sh master --diff-filter=MR --error-exitcode=0 || exit
+		./dist/tools/cppcheck/check.sh master --diff-filter=AC || exit
```

Comment:
The static analysis does not really depend on "$BUILDTEST_MCU_GROUP" == "x86", maybe you could factor it into either it's own group, or have it executed in a different fashion?